package.json

```diff
@@ -32,10 +32,10 @@
     "definition": "dist/index.d.ts"
   },
   "dependencies": {
+    "@anthropic-ai/claude-agent-sdk": "0.1.65",
```

Comment:
Not even patch updates? Like: `~0.1.65`

If not maybe we gotta run `npm audit` in case they do release patches for vulnerabilities